### PR TITLE
YBRO 6 cards

### PR DESCRIPTION
**Pull request by Simisays**
Should be good to merge

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/forgefire_automaton.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Name:Forgefire Automaton
+ManaCost:7
+Types:Artifact Creature Construct
+PT:8/5
+K:Prototype:2 B:3:2
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | Execute$ TrigReturn | TriggerDescription$ At the beginning of your upkeep, return target creature card with power less than CARDNAME's power from your graveyard to the battlefield. Its base power perpetually becomes equal to CARDNAME's Power
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/forgefire_automaton.txt:6]: Missing TriggerZones

File: forge-gui/res/cardsfolder/upcoming/penregon_besieged.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Name:Penregon Besieged
+ManaCost:1 B
+Types:Enchantment
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigChoose | TriggerDescription$ At the beginning of your end step, choose a creature with the least toughness among creatures your opponents control. It perpetually gets -1/-1.
+SVar:TrigChoose:DB$ ChooseCard | Choices$ Creature.leastToughness+OppCtrl | ChoiceTitle$ Choose a creature with the least toughness among creatures your opponents control |  Mandatory$ True | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | StaticAbilities$ PerpetualDebuff | Name$ Penregon Besieged's Perpetual Effect | Duration$ Permanent
+SVar:PerpetualDebuff:Mode$ Continuous | Affected$ Card.ChosenCard | AddPower$ -1 | AddToughness$ -1 | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ This creature perpetually gets -1/-1.
+SVar:DBDestroy:DB$ Destroy | Defined$ ChosenCard | NoRegen$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/penregon_besieged.txt:8]: Random line

File: forge-gui/res/cardsfolder/upcoming/great_desert_hellion.txt [hunk 3]
```diff
@@ -0,0 +1,17 @@
+Name:Great Desert Hellion
+ManaCost:1 B R 
+Types:Creature Hellion
+PT:5/5
+K:Starting intensity:1
+K:Menace
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | Execute$ TrigDiscard | TriggerZones$ Battlefield | TriggerDescription$ At the beginning of your upkeep, sacrifice CARDNAME unless you discard a card. If you discard a card this way, perpetually increase Great CARDNAME's intensity by 1.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/great_desert_hellion.txt:7]: `Great CARDNAME's`

File: forge-gui/res/cardsfolder/upcoming/great_desert_hellion.txt [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+SVar:DBSacrifice:DB$ Sacrifice | Defined$ Self | ConditionDefined$ Remembered | ConditionPresent$ Card | ConditionCompare$ EQ0 | SubAbility$ DBIncrease
+SVar:DBIncrease:DB$ Intensify | ConditionDefined$ Remembered | ConditionPresent$ Card | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Any | ValidCard$ Card.Self | Execute$ TrigDiscardBis | TriggerDescription$ When CARDNAME leaves the battlefield, you may discard your hand. If you do, Draw cards equal to CARDNAME's intensity. 
+SVar:TrigDiscardBis:DB$ Discard | Optional$ True | Mode$ Hand | Defined$ You | SubAbility$ DBDraw
+SVar:DBDraw:DB$ Draw | Defined$ You | NumCards$ X
+SVar:X:TriggeredCard$Intensity
+DeckHas:Ability$Discard|Sacrifice
+Oracle:Starting intensity 1\nMenace\nAt the beginning of your upkeep, sacrifice Great Desert Hellion unless you discard a card. If you discard a card this way, perpetually increase Great Desert Hellion's intensity by 1.\nWhen Great Desert Hellion leaves the battlefield, you may discard your hand. If you do, Draw cards equal to Great Desert Hellion's intensity.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/great_desert_hellion.txt:17]: `Draw` in middle of sentence (also line 12)

**Comment by Simisays** [comment]
Moved Great Desert Hellion to a separate PR so this can be merged

File: forge-gui/res/cardsfolder/upcoming/forgefire_automaton.txt [hunk 5]
```diff
@@ -0,0 +1,13 @@
+Name:Forgefire Automaton
+ManaCost:7
+Types:Artifact Creature Construct
+PT:8/5
+K:Prototype:2 B:3:2
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigReturn | TriggerDescription$ At the beginning of your upkeep, return target creature card with power less than CARDNAME's power from your graveyard to the battlefield. Its base power perpetually becomes equal to CARDNAME's Power
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/forgefire_automaton.txt:6]: power.

small p and full stop. Same for Oracle line

File: forge-gui/res/cardsfolder/upcoming/forgefire_automaton.txt [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+SVar:TrigReturn:DB$ ChangeZone |  ValidTgts$ Creature.powerLTX+YouOwn | RememberChanged$ True | TgtPrompt$ Select target creature card with lesser power | Origin$ Graveyard | Destination$ Battlefield | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | Name$ Forgefire Automaton's Perpetual Effect | RememberObjects$ Remembered | StaticAbilities$ PerpetualStatic | ImprintCards$ Self | Duration$ Permanent
+SVar:PerpetualStatic:Mode$ Continuous | Affected$ Card.IsRemembered | SetPower$ Y | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ The base power of this card perpetually becomes equal to Forgefire Automaton's Power
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/forgefire_automaton.txt:9]: Same as above

File: forge-gui/res/cardsfolder/upcoming/warzone_duplicator.txt [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+SVar:PerpetualAbility:Mode$ Continuous | AddStaticAbility$ SpendAnyMana | Affected$ Card.IsRemembered | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ The conjured card perpetually gains "You may spend mana as though it were mana of any color to cast this spell."
+SVar:SpendAnyMana:Mode$ Continuous | Affected$ Card.Self | EffectZone$ All | AffectedZone$ Stack | AddHiddenKeyword$ May spend mana as though it were mana of any color to cast CARDNAME | Description$ You may spend mana as though it were mana of any color to cast this spell.
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:X:TriggeredCard$CardPower
+DeckHints:Color$Blue
+Oracle:Prototype {3}{U} — 3/3 (You may cast this spell with different mana cost, color, and size. It keeps its abilities and types.)\n\nWhen Warzone Duplicator enters the battlefield, return target creature an opponent controls with power less than Warzone Duplicator's power to it's owner's hand. If that creature wasn't a token, conjure a duplicate of it into your hand. It perpetually gains "You may spend mana as though it were mana of any color to cast this spell".
```
[the remark this answers is not in the document]
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/warzone_duplicator.txt:15]: thanks for the comments !

File: forge-gui/res/cardsfolder/upcoming/forgefire_automaton.txt [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+SVar:TrigReturn:DB$ ChangeZone |  ValidTgts$ Creature.powerLTX+YouOwn | RememberChanged$ True | TgtPrompt$ Select target creature card with lesser power | Origin$ Graveyard | Destination$ Battlefield | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | Name$ Forgefire Automaton's Perpetual Effect | RememberObjects$ Remembered | StaticAbilities$ PerpetualStatic | ImprintCards$ Self | Duration$ Permanent
+SVar:PerpetualStatic:Mode$ Continuous | Affected$ Card.IsRemembered | SetPower$ Y | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ The base power of this card perpetually becomes equal to Forgefire Automaton's power
+SVar:X:Count$CardPower
+SVar:Y:Imprinted$CardPower
+DeckHints:Ability$Graveyard|Discard|Mill & Color$Black
+Oracle:Prototype {2}{B} — 3/2 (You may cast this spell with different mana cost, color, and size. It keeps its abilities and types.)\nAt the beginning of your upkeep, return target creature card with power less than Forgefire Automaton's power from your graveyard to the battlefield.\nIts base power perpetually becomes equal to Forgefire Automaton's power
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/forgefire_automaton.txt:13]: Still missing the . on lines 6, 9 and 13 :P

Comment (paulsnoops) [reply forge-gui/res/cardsfolder/upcoming/forgefire_automaton.txt:13]: Oh also, the \n between battlefield. and Its shouldn't be there, it's part of the same paragraph

File: forge-gui/res/cardsfolder/upcoming/forgefire_automaton.txt [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+SVar:TrigReturn:DB$ ChangeZone |  ValidTgts$ Creature.powerLTX+YouOwn | RememberChanged$ True | TgtPrompt$ Select target creature card with lesser power | Origin$ Graveyard | Destination$ Battlefield | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | Name$ Forgefire Automaton's Perpetual Effect | RememberObjects$ Remembered | StaticAbilities$ PerpetualStatic | ImprintCards$ Self | Duration$ Permanent
+SVar:PerpetualStatic:Mode$ Continuous | Affected$ Card.IsRemembered | SetPower$ Y | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ The base power of this card perpetually becomes equal to Forgefire Automaton's power.
+SVar:X:Count$CardPower
+SVar:Y:Imprinted$CardPower
+DeckHints:Ability$Graveyard|Discard|Mill & Color$Black
+Oracle:Prototype {2}{B} — 3/2 (You may cast this spell with different mana cost, color, and size. It keeps its abilities and types.)\nAt the beginning of your upkeep, return target creature card with power less than Forgefire Automaton's power from your graveyard to the battlefield.Its base power perpetually becomes equal to Forgefire Automaton's power.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/forgefire_automaton.txt:13]: battlefield.Its

Needs space

**Review by paulsnoops — APPROVED** [review]

**Pull request merged**